pkg/chartutil/create.go

```diff
@@ -36,12 +36,37 @@ const (
 	ChartsDir = "charts"
 	// IgnorefileName is the name of the Helm ignore file.
 	IgnorefileName = ".helmignore"
+	// DeploymentName is the name of the example deployment file.
+	DeploymentName = "deployment.yaml"
+	// ServiceName is the name of the example service file.
+	ServiceName = "service.yaml"
```

Comment:
Yeah, just wanted to point it out. I'm fine with either as long as there is a standard.